packages/@react-spectrum/calendar/src/CalendarBase.tsx

```diff
@@ -45,7 +45,8 @@ export function CalendarBase<T extends CalendarState | RangeCalendarState>(props
     prevButtonProps,
     errorMessageProps,
     calendarRef: ref,
-    visibleMonths = 1
+    visibleMonths = 1,
+    firstDayOfWeek = 'sun'
```

Comment:
i thought this was determined by locale/calendar system? do we need a default here?

Reply:
I guess with the current implementation, `'sun'` essentially represents 0 offset from the locale's first day of the week. For France, this would resolve to Monday. I'll update this to map to Sunday regardless of locale instead.